ui/css/module/_footnotes.scss

```diff
@@ -40,8 +31,9 @@ $triangle-border-vertical-offset: ($triangle-dimensions-value + $footnote-vertic
     color: $color-white;
 
     &::before {
-      @include footnote-border-triangle-up();
-      top: $triangle-border-vertical-offset;
+      $triangle-dimensions: 18px;
+      @include footnote-border-triangle-up($triangle-dimensions);
+      top: $triangle-dimensions + $footnote-vertical-margin;
```

Comment:
Im guessing you tested this, but we need to also subtract for the two pixel borders.

Reply:
I removed the borders in the second commit because they _aren't_ lined up now.

## Current Demo
<img width="292" alt="screen shot 2017-12-01 at 5 40 44 pm" src="https://user-images.githubusercontent.com/326918/33506663-e48b645c-d6be-11e7-9d5f-fc83b8c7ca11.png">

## With change
<img width="358" alt="screen shot 2017-12-01 at 5 42 02 pm" src="https://user-images.githubusercontent.com/326918/33506683-f912294c-d6be-11e7-9c19-48a08193b2c6.png">

That said, it's slightly different in firefox:

<img width="267" alt="screen shot 2017-12-01 at 5 42 49 pm" src="https://user-images.githubusercontent.com/326918/33506706-0cc3fc72-d6bf-11e7-8329-968a7d1e3b10.png">

Reply:
👍

Reply:
I wonder if the recent change to the citation text's font size caused the problem?